applications_tests/lethe-fluid/solid_thermocapillary_droplet.prm

```diff
@@ -130,48 +130,48 @@ end
 
 subsection boundary conditions heat transfer
   set number = 4
-    subsection bc 0
-      set id = 0
-      set type	      = temperature
-      subsection value
-        set Function expression = 0
-      end
-    end
-    subsection bc 1
-      set id = 1
-      set type	      = temperature
-      subsection value
-        set Function expression = 6
-      end
-    end
-    subsection bc 2
-      set id = 2
-      set type	      = noflux
+  subsection bc 0
+    set id   = 0
+    set type = temperature
+    subsection value
+      set Function expression = 0
     end
-    subsection bc 3
-      set id = 3
-      set type	      = noflux
+  end
+  subsection bc 1
+    set id   = 1
+    set type = temperature
+    subsection value
+      set Function expression = 6
     end
+  end
+  subsection bc 2
+    set id   = 2
+    set type = noflux
+  end
+  subsection bc 3
+    set id   = 3
+    set type = noflux
+  end
 end
 
 #---------------------------------------------------
 # Post-processing
 #---------------------------------------------------
 
 subsection post-processing
-  set verbosity            = verbose
-  set calculate barycenter = true
+  set verbosity                   = verbose
+  set calculate barycenter        = true
+  set calculate mass conservation = false
 end
 
-
 #---------------------------------------------------
 # FEM
 #---------------------------------------------------
 
 subsection FEM
-  set velocity order = 1
-  set pressure order = 1
-  set VOF order      = 1
+  set velocity order    = 1
+  set pressure order    = 1
+  set VOF order         = 1
   set temperature order = 1
```

Comment:
Do you need to explicitely add those?  I don't mind but it's really just a question.